src/unix/notbsd/linux/mod.rs

```diff
@@ -185,6 +187,29 @@ s! {
     }
 }
 
+f! {
+    pub fn CPU_ZERO(cpuset: &mut cpu_set_t) -> () {
+        for slot in cpuset.bits.iter_mut() {
+            *slot = 0;
+        }
+    }
+
+    pub fn CPU_SET(cpu: usize, cpuset: &mut cpu_set_t) -> () {
+        let (idx, offset) = (cpu / mem::size_of::<::c_ulong>(), cpu % mem::size_of::<::c_ulong>());
```

Comment:
Just to hopefully be a little more resilient about sizes in the future, instead of `mem::size_of::<::c_ulong>()` could this be:

```rust
let size = mem::size_of_val(&cpuset.bits[0]);
let (idx, offset) = (cpu / size, cpu % size);
// ...
```